_posts/blog/2016-27-06-holacrcy-part-1.md

```diff
@@ -0,0 +1,96 @@
+automated, robot driven assembly lines in car production. Is completely impossible in an agile fast 
+changing software development process. 
+What could be the alternative? Anarchy, base democratic decisions or maybe hire an almighty genius who 
+knows everything?
+The answer that Holacracy provides is an accurate set of rules to shift the power to a constitution 
+which is binding for every member of the organization. As like in every other constitution driven 
+culture, this brings the freedom to make your own decisions as long as you stick to the rules. 
+The provocative title of this section could suggest that Holacracy isn’t a hierarchical system - far 
+from it! There is still someone or even a group of people who makes decisions for the whole company, a 
+bit like a government may decide to build an airport, but all the practical choices are made by someone 
+who actually knows how to construct a runway or a hangar (maybe a bad example for a Berlin based company :-) )
+The main purpose of the constitution is to provide “roles” (who is accountable for what) and governance processes 
+(fix or improve the roles).
+
+
+### Role vs. Soul vs. Circle
+The definition of a role consists of three elements:
+
+ * A purpose 
+ * Domains to control
+ * Set of accountabilities and corresponding domains
+ Only the purpose is mandatory, the accountabilities and domains can evolve over time. As an example you can have a look at the developer role from GlassFrog - 
+ which is a web based tool developed by HolacracyOne. 
+ 
+**Developer**
+**Purpose**
+Satisfy the customer, with Product Vision as proxy, through early and continuous delivery of truly valuable software
+
+**Accountabilities**
+
+ * Implementing GlassFrog user stories, bug fixes, and chores in alignment with: 
+ * development practices set by Scrum Master 
+ * implementation standards set by Quality Guardian
+ * Testing new functionality before each production deployment
+ * Providing input into estimation at the request of Scrum Master
+ * Sharing development challenges with Product Vision, API Product Designer, and Developer as challenges arise
+ * Demonstrating working software to Product Vision regularly and frequently, and integrating feedback surfaced by Product Vision
+ 
+A definition like this makes it explicit what you can expect from a developer and on the other hand a dev 
+knows what she/he is accountable for. 
+But as the title of this sections says, there is a big difference between the persons who are working in 
+an organization and the roles they fill. Decoupling these two elements allows improving the culture of 
+an organization drastically. Conflicts in an organization are based on clashes on roles that are 
+involved but they are mistaken them for clashes of the people who are filling the roles. This makes the 
+conflicts personal and we try to resolve them by smoothing the relationships and miss the opportunity 
```

Comment:
...and we try to resolve them by settling a personal dispute rather than resolving the underlying issues in the organizational structure.

(paragraph)